clientv3/integration/cluster_test.go

```diff
@@ -272,12 +272,13 @@ func TestMemberPromote(t *testing.T) {
 
 	// retry until promote succeed or timeout
 	timeout := time.After(5 * time.Second)
+	promLoop:
 	for {
 		select {
 		case <-time.After(500 * time.Millisecond):
 		case <-timeout:
 			t.Errorf("failed all attempts to promote learner member, last error: %v", err)
-			break
+			break promLoop
```

Comment:
Good catch! Can we use t.Fatalf here?

Reply:
Chnage to t.Fataf, in this case, the `break` is no need anymore